grails-app/views/profile/publication.gsp

```diff
@@ -0,0 +1,84 @@
+<%--
+ Created by Temi Varghese on 14/07/15.
+--%>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="layout" content="${grailsApplication.config.layout}"/>
+    <title>Publication</title>
+    <style rel="stylesheet">
+    .selected {
+        background-color: white;
+    }
+
+    .spacing {
+        min-height: 50px;
+    }
+    </style>
+
+    <r:require module="profiles"/>
+
+</head>
+
+<body>
+<div ng-controller="DoiController as doiCtrl" ng-init="doiCtrl.init(${publication})">
+
+    <ol class="breadcrumb" ng-cloak>
+        <li><a class="font-xxsmall" href="${request.contextPath}/">Profile Collections</a></li>
+        <li><a class="font-xxsmall"
+               href="${request.contextPath}/opus/{{doiCtrl.profile.opusId}}">{{doiCtrl.profile.opusName}}</a>
+        </li>
+        <li class="font-xxsmall active">{{doiCtrl.profile.scientificName}}</li>
+    </ol>
+
+    <div class="row">
+        <div class="col-sm-12 selected padding-bottom-1">
+            <h2 class="h1" ng-show="doiCtrl.selectedPublication.title">
+                {{doiCtrl.selectedPublication.title}}
+                <span ng-show="doiCtrl.selectedPublication.version">&nbsp;v. {{doiCtrl.selectedPublication.version}}</span>
+            </h2>
+
+            <div class="citation">{{doiCtrl.selectedPublication.authors}}, published on {{doiCtrl.selectedPublication.publicationDate | date:"dd/MM/yyyy HH:mm"}}</div>
+
+            <p class="" ng-show="doiCtrl.selectedPublication.authors">
```

Comment:
Don't need this - it's just a link back to this page...

Replace it with a simple instruction statement: "Click the download button below to download the publication."